packages/monaco-graphql/package.json

```diff
@@ -27,11 +27,11 @@
   },
   "devDependencies": {
     "graphql": "experimental-stream-defer",
-    "monaco-editor": "^0.20.0",
+    "monaco-editor": "^0.27.0",
     "vscode-languageserver-types": "^3.15.1"
   },
   "peerDependencies": {
     "graphql": ">= v14.5.0 <= 15.5.0",
-    "monaco-editor": "^0.20.0"
+    "monaco-editor": "^0.27.0"
```

Comment:
this is the only thing that needs changed, my bad overlooked this before. this is still compatible with at least 0.20.0 if not earlier. we should test with a version matrix perhaps

Reply:
you mean, we need to undo all other file changes, right?  @acao

Reply:
@acao I have reverted it. please review it.

Reply:
Fantastic, thank you!